### PR TITLE
feat: add Simplified Chinese supported

### PR DESCRIPTION
**Pull request by xfqz86**
feat: auto set default language based on browser language
fix: Text description error in ImageEditor.svelte

### CODE REVIEW

**Comment by MultiMote** [comment]
Thank you so much for your work!

Unfortunately, there are some problems

image

you can reproduce it by running `yarn sv-check`

File: niimblue/src/utils/i18n.ts [hunk 1]
```diff
@@ -1,18 +1,35 @@
 import { derived, writable } from "svelte/store";
-import { langPack } from "../lang_pack";
+import type { translationKeys, supportedLanguages } from "../lang_pack"
+import { languagesMaps, langPack } from "../lang_pack";
 
-export const locales = Object.keys(langPack);
+function brewserLanguage2supportedLanguagesKeys(brewserLanguage: string): supportedLanguages {
```
Comment (MultiMote) [inline niimblue/src/utils/i18n.ts:5]: Typo

**Review by MultiMote — CHANGES_REQUESTED** [review]

File: niimblue/src/utils/i18n.ts (hunk 1 above)
Comment (xfqz86) [reply niimblue/src/utils/i18n.ts]: fixed

**Comment by xfqz86** [comment]
> Thank you so much for your work!
> 
> Unfortunately, there are some problems
> 
> image
> 
> you can reproduce it by running `yarn sv-check`

fixed

**Pull request merged**